### PR TITLE
S05 car page

### PR DESCRIPTION
**Pull request by ItsMatDev**

### CODE REVIEW

File: frontend/src/assets/cars/PeugeotE2082.png
Comment (Bravax) [inline frontend/src/assets/cars/PeugeotE2082.png:1]: Remove duplicate car

File: frontend/src/components/CarShow/AddCarModal/AddCarModal.scss [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+.Input {
+  width: 100%;
+  flex: 1;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 4px;
+  padding: 0 10px;
+  font-size: 15px;
```
Comment (Bravax) [inline frontend/src/components/CarShow/AddCarModal/AddCarModal.scss]: no px to change

File: frontend/src/components/CarShow/AddCarModal/AddCarModal.scss [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  line-height: 1;
+  color: var(--violet-11);
+  box-shadow: 0 0 0 1px var(--violet-7);
+  height: 35px;
```
Comment (Bravax) [inline frontend/src/components/CarShow/AddCarModal/AddCarModal.scss]: no px to change

File: frontend/src/components/CarShow/AddCarModal/AddCarModal.scss [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+.DialogContent {
+  background-color: $colorLight;
+  border-radius: 6px;
+  box-shadow: $boxShadow;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 90vw;
+  max-width: 450px;
```
Comment (Bravax) [inline frontend/src/components/CarShow/AddCarModal/AddCarModal.scss]: no px to change

File: frontend/src/components/CarShow/AddCarModal/AddCarModal.scss [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+  margin: 0;
+  font-weight: 500;
+  color: $colorDark;
+  font-size: 17px;
+}
+
+.DialogDescription {
+  margin: 10px 0 20px;
+  color: $colorDark;
+  font-size: 15px;
```
Comment (Bravax) [inline frontend/src/components/CarShow/AddCarModal/AddCarModal.scss]: no px to change

File: frontend/src/components/CarShow/AddCarModal/AddCarModal.scss [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+}
+
+.buttonAddCar {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  background: $gradient180deg;
+  border-radius: $borderButton;
+  padding: 0 15px;
+  font-size: 15px;
```
Comment (Bravax) [inline frontend/src/components/CarShow/AddCarModal/AddCarModal.scss]: no px to change

File: frontend/src/components/CarShow/AddCarModal/AddCarModal.scss [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+  line-height: 1;
+  font-weight: 500;
+  height: 35px;
```
Comment (Bravax) [inline frontend/src/components/CarShow/AddCarModal/AddCarModal.scss]: no px to change

**Review by SebSamPro — APPROVED** [review]

File: frontend/src/components/CarShow/AddCarModal/AddCarModal.scss (hunk 1 above)
Comment (ItsMatDev) [reply frontend/src/components/CarShow/AddCarModal/AddCarModal.scss]: fixed font-size to responsive value rem

File: frontend/src/components/CarShow/AddCarModal/AddCarModal.scss (hunk 4 above)
Comment (ItsMatDev) [reply frontend/src/components/CarShow/AddCarModal/AddCarModal.scss]: fixed font-size to responsive value rem

File: frontend/src/components/CarShow/AddCarModal/AddCarModal.scss (hunk 5 above)
Comment (ItsMatDev) [reply frontend/src/components/CarShow/AddCarModal/AddCarModal.scss]: fixed font-size to responsive value rem

File: frontend/src/components/CarShow/AddCarModal/AddCarModal.scss (hunk 2 above)
Comment (ItsMatDev) [reply frontend/src/components/CarShow/AddCarModal/AddCarModal.scss]: this height need to be fixe

File: frontend/src/components/CarShow/AddCarModal/AddCarModal.scss (hunk 3 above)
Comment (ItsMatDev) [reply frontend/src/components/CarShow/AddCarModal/AddCarModal.scss]: this height need to be fixe

File: frontend/src/components/CarShow/AddCarModal/AddCarModal.scss (hunk 6 above)
Comment (ItsMatDev) [reply frontend/src/components/CarShow/AddCarModal/AddCarModal.scss]: this height need to be fixe

**Review by Bravax — APPROVED** [review]

**Review by JulienOza — APPROVED** [review]

File: frontend/src/components/CarShow/CarShow.jsx [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+          className="nextCar"
+          type="button"
+          onClick={() => setcounterCar(counterCar - 1)}
+        >
+          <img className="nextCarButton nextCarMirror" src={button} alt="" />
+          <p className="nextCarText">Voiture</p>
+          <p className="nextCarText">précédente</p>
+        </button>
+      )}
+      <article className="carCardPlug">
```
Comment (Yoann-Fortin) [inline frontend/src/components/CarShow/CarShow.jsx]: Figure and figcaption is better here

File: frontend/src/components/CarShow/CarShow.jsx [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+        <img className="carPlugType" src={type2} alt="" />
+        <p className="carCardText">Type 2</p>
+      </article>
+      <article className="carCardPlug">
```
Comment (Yoann-Fortin) [inline frontend/src/components/CarShow/CarShow.jsx]: Here too

**Review by Bravax — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**